packages/ui/components/validate-messages/translations/fr.js

```diff
@@ -13,8 +13,8 @@ export default {
     MaxNumber: 'Veuillez indiquer un(e) {fieldName} inférieur(e) à {params}.',
     MinMaxNumber: 'Veuillez indiquer un(e) {fieldName} entre {params.min} et {params.max}.',
     IsDate: ' Veuillez entrer la date (JJ-MM-AAAA).',
-    MinDate: 'Veuillez indiquer un(e) {fieldName} après {params.min, date, YYYYMMDD}.',
-    MaxDate: 'Veuillez indiquer un(e) {fieldName} avant {params.max, date, YYYYMMDD}.',
+    MinDate: 'Veuillez indiquer un(e) {fieldName} après {params, date, YYYYMMDD}.',
+    MaxDate: 'Veuillez indiquer un(e) {fieldName} avant {params, date, YYYYMMDD}.',
```

Comment:
Could you explain what this change is about? I run the tests for the PR without this file modified and all tests are passed as well.